tools/src/main/java/io/github/ygojson/tools/dataprovider/impl/yugipedia/mapper/CardNumberMapper.java

```diff
@@ -42,6 +42,63 @@ public class CardNumberMapper {
 		GeneralMapper.class
 	);
 
+	@Named("unknownToNull")
+	protected String mapUnknownToNull(final String value) {
```

Comment:
unknown keyword should be changed all around to unreleased to be clear of what we mean  here